addons/website_payment/static/src/js/website_payment_form.js

```diff
@@ -7,6 +7,7 @@ import checkoutForm from 'payment.checkout_form';
 checkoutForm.include({
     events: _.extend({}, checkoutForm.prototype.events || {}, {
         'change .o_wpayment_fee_impact': '_onFeeParameterChange',
+        'focus .o_wpayment_fee_impact': '_onFeeParameterChange',
```

Comment:
This may be a bit extreme because it will make *two* RPC calls for every click. What is the 'change' even listener missing, currently?

Reply:
If we keep only the "change", we don't cover the case where the user:
- type a custom amount
- select a predefined amount
- change back to the custom without editing
- submit

In this case, the wrong amount would be donated.

But you are right about the RPC, I will find another solution. Thanks